### ISSUE TITLE
Question: Is there a way to disable Proxyman proxy while keeping app open?

### ISSUE THREAD

**Issue by fishcharlie** [ProxymanApp/Proxyman#784]
Is there a way to disable Proxyman proxy while keeping app open? When clicking the pause button in the top left it pauses recording but keeps the proxy enabled. Is there a similar button to turn on and off the proxy itself without closing the application?

### Proxyman version? (Ex. Proxyman 1.4.3)

Version 2.17.0 (21700)

### macOS Version? (Ex. mac 10.14)

macOS 11.3 Beta (20E5172i)

**Comment by NghiaTranUIT** [comment]
Yes, please Toggle in Tool -> Proxy Setting -> Override Proxy Setting or **⌥⌘O** to toggle on/off the Proxy Config in Network Preference 👍

**Comment by NghiaTranUIT** [comment]
@fishcharlie  It seems that there are many users, who ask the same question, are not aware of "⌥⌘O to toggle on/off the Proxy". They often click on the pause button and wonder how to turn off the Proxy.

I guess I will add a small note in the Status Bar if they try to Pause the app 😄 

<img width="1121" alt="Screen_Shot_2021-02-12_at_11_17_19" src="https://user-images.githubusercontent.com/5878421/107729664-16944c80-6d24-11eb-9df3-5631b00ca355.png">
